### ISSUE TITLE
sudo sftp

### ISSUE THREAD

**Issue by ceceghini** [h3imdall/ftp-remote-edit#405]
Is possible to configureIt is possible to configure the sftp connection to be able to use sudo in the remote environment similar to the following command:
sftp -s "sudo / usr / lib / openssh / sftp-server" targethost.fqdn

**Comment by jdsjdk** [comment]
Yes, I find this troubling when I have this very helpful tool available to me, but then I am **forced** back to using the terminal because I get "permission denied".  So my only resource is to use sudo commands within the terminal again.  It would be **incredibly** helpful to code **anywhere** in the server.  _Including_ config files.
